### DISCUSSION TITLE
Pusher SDK Client Connection failing

### DISCUSSION THREAD

**Discussion by briangwaltney** [soketi/pws#791] [Q&A]
Hello,

I'm trying to get soketi working on a new app. I'm migrating from Pusher and I believe I have the backend setup correctly. Server triggers are receiving a 200 response, but I can't get the client to connect properly.

```typescript
export const pusher = new Pusher(env.NEXT_PUBLIC_PUSHER_KEY, {
  cluster: env.NEXT_PUBLIC_PUSHER_CLUSTER,
  wsHost: env.NEXT_PUBLIC_PUSHER_HOST,
  wsPort: parseInt(env.NEXT_PUBLIC_PUSHER_PORT),
  enabledTransports: ["ws", "wss"],
  forceTLS: false,
  enableStats: false,
  
});
```

Above is my client creation config with all the same values as my backend.

This is the config output on the client:

```typescript
activityTimeout: 120000
channelAuthorizer: ƒ (params, callback)
cluster: ""
enableStats: false
enabledTransports: (2) ['ws', 'wss']
httpHost: "sockjs.pusher.com"
httpPath: "/pusher"
httpPort: 80
httpsPort: 443
pongTimeout: 30000
statsHost: "stats.pusher.com"
unavailableTimeout: 10000
useTLS: false
userAuthenticator: ƒ (params, callback)
wsHost: "127.0.0.1"
wsPath: ""
wsPort: 6001
wssPort: 443
```

It seems like it's still trying to connect through Pusher's host. Is that what I want? When I block out the `wsHost` and `wsPort` it does successfully connect via Pusher, but doesn't receive messages from my new server (correctly, of course).

What am I missing in the setup? How do I get the frontend client to connect to the new soketi server?

**Comment by heribertolord96** [comment]
I have similar problem .
It fails at build an deploy. 
In mi case i creaded an laravel API as backend for emmit events and it works fine at upload to production , But the js client app cant connect to socketi server in production , but works in local . Maybe syntax error ocurs when yarn build
